node/tests/swamp/swamp.go

```diff
@@ -296,3 +297,14 @@ func (s *Swamp) remove(rn *node.Node, sn []*node.Node) ([]*node.Node, error) {
 	}
 	return sn, nil
 }
+
+// Disconnect allows to break a connection between two peers without any possibility to re-establish it.
```

Comment:
We technically can reconnect them by linking and connecting peers back, and I suggest the addition of another symmetric `Connect` method that does this. The comment can mention that this is __hard__ disconnect which can reconnect with respective `Connect`, peers themselves would not be able to reconnect.

Reply:
added